packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -55,6 +56,15 @@ export class TriggerManager extends Component {
     }
   }
 
+  componentDidMount() {
+    try {
+      CozyFolder.registerClient(this.props.client)
```

Comment:
It feels weird to have this here. We should find something better than that IMHO.